lib/nimble.phx.gen.template/addons/elixir_version.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Nimble.Phx.Gen.Template.Addons.ElixirVersion do
+  use Nimble.Phx.Gen.Template.Addon
+
+  @impl true
+  def do_apply(%Project{} = project, _opts) do
+    project
+    |> copy_files()
+    |> edit_files()
+  end
+
+  defp copy_files(project) do
+    Generator.copy_file([{:text, ".tool-versions", ".tool-versions"}])
+
+    project
+  end
+
+  defp edit_files(project) do
+    Generator.replace_content(
+      "mix.exs",
+      """
+            elixir: "~> 1.7",
+      """,
+      """
+            elixir: "~> 1.11",
```

Comment:
What do you think of having a constant defined in `Nimble.Phx.Gen.Template.Project` 💭 The advantage is that it would not be hidden deep in an addon.

Reply:
Moved to Project in https://github.com/nimblehq/elixir-templates/pull/36/commits/ba94b5ddb712ae8475daec36ff238b86460e1b87 @olivierobert